src/App.js

```diff
@@ -1,2 +1,23 @@
-const App = () => <h1 className="red">React App</h1>;
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
```

Comment:
You successfully created a single page application (SPA) using react-router-dom to serve each page without reloading 🙌🔥

Reply:
👍